.devcontainer/docker-compose.yml

```diff
@@ -61,7 +61,7 @@ services:
       - 5432:5432
     volumes:
       - postgres-data:/var/lib/postgresql/data
-      - ${PWD}/postgresql/init.sql:/docker-entrypoint-initdb.d/init.sql
+      - ./postgresql:/docker-entrypoint-initdb.d
```

Comment:
So you can't mount a single file, you need to mount the whole folder as a volume ?